platform/android/export/export.cpp

```diff
@@ -1764,16 +1764,16 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 
 		if (!FileAccess::exists(js)) {
 			valid = false;
-			err += TTR("OpenJDK jarsigner not configured in the Editor Settings.") + "\n";
+			err += TTR("OpenJDK 8 jarsigner not configured in the Editor Settings.") + "\n";
 		}
 
 		String dk = p_preset->get("keystore/debug");
```

Comment:
We don't know if the user will be exporting in release mode at this point. How could error checking be improved to take into account a situation where only a release keystore is defined in the export preset, but no default keystore is defined in the Editor Settings? Right now, this will be considered an error.

Reply:
I think it's fine to have an error in this case, given the way things are currently set up for debug and release.